modules/sticky-table-headers/main/index.coffee

```diff
@@ -258,28 +274,33 @@ class StickyTableHeaders
 
       topLeftTable = topLeftHead.append tableClone.clone(true)
 
-      realNodes = table.select('thead').selectAll('tr').select('th, td').nodes
-      clonedNodes = topLeftTable.select('thead').selectAll('tr').select('th, td').nodes
-
-      for i in [0...realNodes.length]
-        cloneEvents(realNodes[i], clonedNodes[i])
-        hx.select(realNodes[i]).classed('hx-sticky-table-invisible', true)
+      realNodes = getChildrenFromTable table, false, true
+      clonedNodes = getChildrenFromTable topLeftTable, false, true
 
+      createStickyHeaderNodes realNodes, clonedNodes
 
     topHead?.style('height', offsetHeight + 'px')
       .style('width', wrapperBox.width + 'px')
       .style('left', offsetWidth + 'px')
+      .selectAll('.hx-sticky-table-invisible')
```

Comment:
Just saw your second comment. 
topHead is a clone of the table with a 'mask' applied so you only see the header, leftHead is the same but for the first column, topLeftHead is the same but for the top left cell.
They are complete clones as it's the only way to guarantee the tables all have the same column/row widths and heights without having to fix the sizes for every cell.
The three elements are added by sticky table headers and shouldn't have any nested functionality so classing here shouldn't be a problem.